lib/datasource/gitlab-packages/readme.md

```diff
@@ -0,0 +1,38 @@
+[GitLab Packages API](https://docs.gitlab.com/ee/api/packages.html) supports looking up package versions from [all types of packages registry supported by GitLab](https://docs.gitlab.com/ee/user/packages/package_registry/index.html) and can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
+
+To specify which specific repository should be queried when looking up a package, the `registryUrl` has to be set like this: `https://gitlab.com/user/project`.
```

Comment:
```suggestion
To specify which specific repository should be queried when looking up a package, the `registryUrl` has to be set following this format: `https://gitlab.com/user/project`.
```

I think we're trying to explain that you should follow a specific format when setting the `registryUrl`.